v0/util/enum/message.js

```diff
@@ -0,0 +1,9 @@
+const Message = {
```

Comment:
I think this error constant object can be kept under `utils/index.js` instead of creating a new file